libs/mstable/flatcoin/src/components/TradingPanel/TradingButton/StableTradingButton.tsx

```diff
@@ -28,11 +29,7 @@ const useStableTradingButton = () => {
     slippage,
     reset,
   } = useFlatcoinTradingState();
-  const lowerThanKeeperFee =
-    isDeposit &&
-    new BigNumber(sendToken.value || '0')
-      .shiftedBy(sendToken.decimals)
-      .lte(keeperFee.exact.toString());
+  const tradingAmountLowerThanKeeperFee = receiveToken.value === '0';
```

Comment:
is it worth to compare like: 
`withreceiveToken.value < keeperFee.exact.toString()`